anime_downloader/sites/kissanimex.py

```diff
@@ -0,0 +1,41 @@
+from anime_downloader.sites import helpers
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+
+class KissAnimeX(Anime, sitename = 'kissanimex'):
+    sitename = "kissanimex"
+
+    @classmethod
+    def search(cls, query):
+        url = 'https://kissanimex.com/search?url=search'
+        r = helpers.get(url, params={'q': query})
+        soup = helpers.soupify(r.text)
+        items = soup.find_all('td')
```

Comment:
I'd recommend using CSS selectors more extensivley instead of doing multiple finds
```suggestion
        items = soup.select('td > a')
```